elmslie-core/src/main/java/vivid/money/elmslie/core/config/ElmslieConfig.kt

```diff
@@ -14,6 +15,8 @@ object ElmslieConfig {
 
     @Volatile private var _shouldStopOnProcessDeath: Boolean = true
 
+    @Volatile private var _globalStoreListeners: Set<StoreListener<Any, Any, Any, Any>>? = null
```

Comment:
maybe make it with just emptySet?

Reply:
why? null is more explicit

Reply:
its a collection, no need for making this nullable, already has empty state

Reply:
lol) what kind of iterations on the empty set?)

Reply:
null check vs iteration on an empty set - what is more explicit?

Reply:
I don't want to argue about it, it's not a critical thing
since you insist I changed it